### ISSUE TITLE
Question about calculating the discriminator loss.

### ISSUE THREAD

**Issue by syguan96** [LeslieZhoa/GFPGAN-1024#10]
Hi @LeslieZhoa , thanks for sharing the code organized with a more clean structure.  When reading the code, I'm confused by one code block (shown below). When computing d_loss of eye discriminators, you use `PartGANLoss` to calculate the real_score but use `GANLoss` to calculate the fake score. Could you please explain why not using the same criterion?

```
            # left eye
            fake_d_pred,_ = self.left_eye_d(fake_left_eyes.detach())
            real_d_pred,_ = self.left_eye_d(real_left_eyes)
            real_left_eye_score = self.PartGANLoss(real_d_pred,True, is_disc=True)
            fake_left_eye_score = self.GANLoss(fake_d_pred, False, is_disc=True)
            l_d_left_eye = real_left_eye_score + fake_left_eye_score 
            l_d += l_d_left_eye
```
